And for 4.08 OCaml version (and upcoming 4.09 as well) there is a dependency restriction:
```
+opam depext -u hardcaml-yosys
# Detecting depexts using vars: arch=x86_64, os=linux, os-distribution=ubuntu, os-family=debian
[ERROR] No solution for hardcaml-yosys: The following dependencies couldn't be met:
          - hardcaml-yosys → hardcaml < 2.0.0 → ocaml < 4.08.0
              base of this switch (use `--unlock-base' to force)
          - hardcaml-yosys → hardcaml < 2.0.0 → js_of_ocaml < 3.0 → ocaml < 4.05.0
              base of this switch (use `--unlock-base' to force)
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -7,7 +7,6 @@ env:
     - OPAMPKG=hardcaml-yosys
     - DEPPKGS=""
   matrix:
-    - OCAML_VERSION=4.02
     - OCAML_VERSION=4.03
     - OCAML_VERSION=4.04
     - OCAML_VERSION=4.05

